### PR TITLE
[search] Support cities postcodes from World.mwm.

### PR DESCRIPTION
**Pull request by tatiana-yan**

### CODE REVIEW

**Comment by tatiana-yan** [comment]
JTMAC

**Comment by tatiana-yan** [comment]
JTMAC

File: search/geocoder.cpp [hunk 1]
```diff
@@ -621,15 +631,9 @@ void Geocoder::FillLocalityCandidates(BaseContext const & ctx, CBV const & filte
...
 {
-  vector<shared_ptr<MwmInfo>> infos;
-  m_dataSource.GetMwmsInfo(infos);
-
-  MwmSet::MwmHandle handle = indexer::FindWorld(m_dataSource, infos);
-  if (handle.IsAlive())
-  {
-    auto context = make_unique<MwmContext>(move(handle));
+  auto context = GetWorldContext(m_dataSource);
+  if (context)
```
Comment (mpimenov) [inline search/geocoder.cpp]: nit:
if (auto context = ...

File: search/search_integration_tests/processor_test.cpp [hunk 2]
```diff
@@ -2230,12 +2230,12 @@ UNIT_CLASS_TEST(ProcessorTest, CityPostcodes)
   SetViewport(m2::RectD(-1, -1, 1, 1));
   {
     Rules rules{ExactMatch(countryId, building4), ExactMatch(countryId, street)};
-    // Test that we not require the building to have a postcode if the city has.
+    // Test that we not require the building to have a postcode if the village has.
```
Comment (mpimenov) [inline search/search_integration_tests/processor_test.cpp]: s/not do not/

File: search/geocoder.cpp (hunk 1 above)
Comment (tatiana-yan) [reply search/geocoder.cpp]: done

File: search/search_integration_tests/processor_test.cpp (hunk 2 above)
Comment (tatiana-yan) [reply search/search_integration_tests/processor_test.cpp]: done

**Pull request merged**